src/SW_netCDF.c

```diff
@@ -69,11 +49,12 @@ static void nc_read_atts(SW_NETCDF* ncInfo, PATH_INFO* PathInfo,
             };
 
     FILE *f;
-    char inbuf[MAX_FILENAMESIZE * 2], value[MAX_FILENAMESIZE * 2]; // * 2 - fit crs_wkt
+    char inbuf[LARGE_VALUE], value[LARGE_VALUE];
     char key[30]; // 30 - Max key size
```

Comment:
in `nc_read_atts()`: `char key[30]; // 30 - Max key size` -- this needs to be `key[35]`; see commit 9903b408e7d0f61a784f20ae4083e9aeef53fa1a